Source/Modeling/Plugins/JoistArea/Installer/Manifest.xml

```diff
@@ -46,8 +46,8 @@
 		<!-- <File Id="RibbonImages" Source="%SourceImages%\*.*" Target="%ExtensionsDir%\Images" Recursive="yes" /> -->
 		<File Id="PluginFiles" Source="%SourceModelPlugins%\*.*" Target="%ModelPluginsDir%" Recursive="yes" />
 		<File Id="TranslationsFiles" Source="%SourceMessages%\*.*" Target="%ModelPluginsDir%" Recursive="yes" />
-		<File Id="GlobalCommon" Source="%SourceCommmon%\*.*" Target="%TargetCommonDir%" Recursive="yes" />
-		<File Id="USARootFiles" Source="%SourceUsaRoot%\*.*" Target="%TargetUsaRoot%" Recursive="yes" />
+		<!-- <File Id="GlobalCommon" Source="%SourceCommmon%\*.*" Target="%TargetCommonDir%" Recursive="yes" /> -->
+		<!-- <File Id="USARootFiles" Source="%SourceUsaRoot%\*.*" Target="%TargetUsaRoot%" Recursive="yes" /> -->
```

Comment:
Why are these now turned off?

Reply:
the folders are empty or non existent in the repo